README.md

```diff
@@ -11,7 +11,7 @@
 Ardupilot is the most advanced, full-featured and reliable open source autopilot software available. It has
 been under development since 2010 by a team of diverse professional engineers and computer scientists. It is
 the only autopilot software capable of controlling almost any vehicle system imaginable, from conventional
-airplanes, multirotors, and helicopters, to boats and even submarines. And now being expanded to feature
+airplanes, multirotors, and helicopters, to boats and even submarines. And is now being expanded to provide
```

Comment:
if we are going to update, might as well make it up to date on vehicle support also...
```suggestion
airplanes, quadplanes, multirotors, and helicopters, to rovers, boats, balancebots, and even submarines. And is now being expanded to provide
```